src/server/routes.pl

```diff
@@ -497,9 +505,9 @@
 
             api_read_document_selector(
                 System_DB, Auth, Path, Graph_Type,
-                Id, Type, Query, Config,
+                Id, Ids, Type, Query, Config,
                 Requested_Data_Version, Actual_Data_Version,
-                cors_json_stream_write_headers_(Request, Actual_Data_Version)
+                routes:cors_json_stream_write_headers_(Request, Actual_Data_Version)
```

Comment:
??

Reply:
yeah i don't know. it started complaining here. slightly cargo cult but what can you do.